src/style.css

```diff
@@ -0,0 +1,79 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url("fonts/Avenir-Book.ttf"), url("fonts/Avenir-Heavy.ttf"), url("fonts/Avenir.ttc");
+}
+
+[data-qa="big"] {
+  font-family: Avenir, Heavy, sans-serif;
+  margin-top: 12px;
+  width: 100%;
+  height: 70px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
```

Comment:
Use CSS variables for repeatable colors.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)